src/main/java/com/communityratesgames/model/GameModel.java

```diff
@@ -1,23 +1,126 @@
 package com.communityratesgames.model;
 
 import java.io.Serializable;
+import java.io.StringReader;
 import java.sql.Timestamp;
-import java.util.List;
 import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
 
 import com.communityratesgames.domain.Game;
 import com.communityratesgames.domain.Platform;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.*;
+
+import javax.json.*;
 
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
 public class GameModel implements Serializable {
 
     private Long id;
     private Timestamp releaseDate;
     private String title;
-    private CompanyModel company;
-    private List<PlatformModel> platforms;
+    private String company;
+    private List<String> platforms;
     private boolean verified;
     private float averageRating;
 
+
+    private JsonObject jsonFromString(String input) {
+        JsonReader jsonReader = Json.createReader(new StringReader(input));
+        JsonObject object = jsonReader.readObject();
+        jsonReader.close();
+
+        return object;
+    }
+
+
+    public GameModel test(String input) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        List<String> list = new ArrayList<>();
+        try {
+            JsonNode root = objectMapper.readTree(input);
+            GameModel gameModel = new GameModel();
+            gameModel.setReleaseDate(Timestamp.valueOf(root.get("releaseDate").asText()));
+            gameModel.setTitle(root.get("title").asText());
+            gameModel.setCompany(root.get("company").asText());
+            JsonNode jn = root.get("platforms");
+            for (JsonNode n : jn
+                 ) {
+                list.add(n.asText());
+            }
+            gameModel.setPlatforms(list);
+
+
+         //   gameModel.setCompany(json.getString("company"));
+           // gameModel.setPlatforms(herp);
+            return gameModel;
+        }
+        catch(Exception e) {
+            e.printStackTrace();
+            return null;
+        }
```

Comment:
Catch more specific exceptions than just `Exception`. It's a bad habit that we must break out of.
Judging from your case, you would want to catch `IOException` and `JsonProcessingException` (See https://fasterxml.github.io/jackson-databind/javadoc/2.0.0/com/fasterxml/jackson/databind/ObjectMapper.html#readTree(byte[])).